Sources/Turf/MultiLineString.swift

```diff
@@ -10,6 +10,10 @@ import CoreLocation
 public struct MultiLineString: Codable, Equatable {
     var type: String = GeometryType.MultiLineString.rawValue
     public var coordinates: [[CLLocationCoordinate2D]]
+    
+    public init(_ coordinates: [[CLLocationCoordinate2D]]) {
```

Comment:
Should there be a more object-oriented representation like `[LineString]`? Not a dealbreaker for this PR, but it seems less than ideal that we can’t better prevent against developers accidentally passing in malformed coordinate arrays that aren’t line strings.

Reply:
Yeah, I was only thinking of a convenience initializer that maps `LineString`s to their `coordinates`es.